src/components/text/index.tsx

```diff
@@ -157,7 +157,6 @@ class Text extends PureComponent<PropsTypes> {
 const styles = StyleSheet.create({
   container: {
     backgroundColor: 'transparent',
-    textAlign: 'left',
```

Comment:
Did you check RTL? 
Also, did you check behavior on Android, it tends to be different sometimes..

Reply:
Actually Android is acting fine with and without this line. Yep, problem in RTL - good catch!